### PR TITLE
Issues and PR templates

### LINKED ISSUE #1125: Issues and PR templates
I have rapidly made a POC of templates for issues and PR, with the hope it can help:
 - formalizing the context, the questions and the needs for issues;
 - clarifying what is proposed for PRs;

Here are my progress for now ( :warning: not ready for a review :warning: ):
- for the issues: https://github.com/fflorent/grist-core/issues/new/choose
- for the PRs (you need to click on "Create Pull Request") : https://github.com/fflorent/grist-core/compare/main...fflorent:grist-core:sponsors-section

Before going further, what do you think of the idea?

### PR DESCRIPTION
**Pull request by fflorent**
## Context

Issues and PRs descriptions sometimes lacks of details we would like to have to better understand the motivations behind.

## Proposed solution

I propose here templates to guide the reporters towards good descriptions.

These templates are heavily inspired from the ones of the [PeerTube](https://github.com/Chocobozzz/PeerTube/) project.

## Related issues

Fixes #1125

## Has this been tested?

- [ ] 👍 yes, I added tests to the test suite
- [ ] 💭 no, because this PR is a draft and still needs work
- [ ] 🙅 no, because this is not relevant here
- [ ] 🙋 no, because I need help

#### Issue first screen

image

#### Bug report

image

#### Installation issue

image

#### Feature request

image

#### PRs

image

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Description
```
[the remark this answers is not in the document]
Comment (fflorent) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🤷💻🤦 Question/Forum
+    url: https://community.getgrist.com/
+    about: You can ask and answer other questions here
+  - name: 💬 Discord
+    url: https://discord.gg/xBjD9PU9
```
Comment (paulfitz) [inline .github/ISSUE_TEMPLATE/config.yml:7]: This shows as invalid https://discord.gg/xBjD9PU9

**Review by paulfitz — COMMENTED** [review]
Very nice @fflorent, thanks for this! One question about a link.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (fflorent) [reply .github/ISSUE_TEMPLATE/config.yml]: Thanks for having check that part, that should be fixed.

I took the link provided here, which seems correct this time: https://support.getgrist.com/newsletters/2023-11/#hang-out-with-us-on-discord

**Review by paulfitz — APPROVED** [review]
Thank you @fflorent !!

**Pull request merged**